pkg/tecdsa/marshaling.go

```diff
@@ -0,0 +1,156 @@
+package tecdsa
```

Comment:
For potential fellow reviewers who has not worked on v1: you want to see https://github.com/keep-network/keep-ecdsa/blob/main/pkg/ecdsa/tss/marshaling.go.